README.md

```diff
@@ -82,197 +80,229 @@ The library comprises two packages:
 
     ```scala 
     @import uk.gov.hmrc.hmrcfrontend.views.html.helpers.HmrcLayout
     @import uk.gov.hmrc.hmrcfrontend.views.config.StandardBetaBanner
     @import views.html.helper.CSPNonce
     @import config.AppConfig
-
+    @import uk.gov.hmrc.anyfrontend.controllers.routes
+    
     @this(hmrcLayout: HmrcLayout, standardBetaBanner: StandardBetaBanner)
 
-    @(
-      pageTitle: String,
-      isWelshTranslationAvailable: Boolean = true)(
-      contentBlock: Html)(
-      implicit request: RequestHeader, messages: Messages)
+    @(pageTitle: String)(contentBlock: Html)(implicit request: RequestHeader, messages: Messages)
 
     @hmrcLayout(
       pageTitle = Some(pageTitle),
-      isWelshTranslationAvailable = isWelshTranslationAvailable,
-      signOutUrl = Some(appConfig.signOutUrl),
-      displayHmrcBanner = true,
+      isWelshTranslationAvailable = true, /* or false if your service has not been translated */
+      serviceUrl = Some(routes.IndexController.index().url),
+      signOutUrl = Some(routes.SignOutController.signOut().url),
       phaseBanner = Some(standardBetaBanner(url = appConfig.betaFeedbackUrl)),
       nonce = CSPNonce.get,
     )(contentBlock)
     ```
-   
+
 1. The parameters that can be passed into the `hmrcLayout` and their default values are as follows:
-    ```scala
-    @(
-      serviceName: Option[String] = None,           // This will be bound to the govukTemplate
-      pageTitle: Option[String] = None,             // This will be bound to the govukTemplate
-      isWelshTranslationAvailable: Boolean = false, // Setting to true will display the language toggle
-      signOutUrl: Option[String] = None,            // Passing value will display the sign out link
-      userResearchBannerUrl: Option[String] = None, // Passing value will display the UserResearchBanner
-      displayHmrcBanner: Boolean = false,           // Setting to true will display the HMRC banner
-      phaseBanner: Option[PhaseBanner] = None,      // Passing value will display alpha or beta banner
-      additionalHeadBlock: Option[Html] = None,     // Passing value will add additional content in head
-      additionalScriptsBlock: Option[Html] = None,  // Passing value will add additional scripts between the footer and end of body
-      nonce: Option[String] = None,                 // Passing value will bind in head and scripts
-      mainContentLayout: Option[Html => Html] = Some(defaultMainContent(_)) // Passing value will allow custom styling of main content
-    )(contentBlock: Html)(implicit request: RequestHeader, messages: Messages)
-    ```
-    
-## Custom layout
 
-If you don't wish to use the `hmrcLayout`, you can create a layout template using `govukLayout` that you can use in all of your service pages. For example:
+   | Parameter                     | Description                                                       | Example                                                   |
```

Comment:
Suggestion: for documenting the parameters of `hmrcLayout` above, we give an example case class. Here, we have a table of parameters. Shall we pick one format for consistency?